### PR TITLE
Upgraded phpunit to allow 5.7 and 6.0

### PR DESCRIPTION
**Pull request by ruudk**
Finishes #45

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -5,3 +5,10 @@
 require __DIR__ . '/../vendor/autoload.php';
 
 AnnotationRegistry::registerLoader('class_exists');
+
+// Polyfill PHPUnit 6.0 both ways
```
Comment (garak) [inline tests/bootstrap.php:9]: Is this really needed? Namespaced TestCase is in phpunit 5.7 already

Comment (ruudk) [reply tests/bootstrap.php:9]: See https://travis-ci.org/SimpleBus/SymfonyBridge/jobs/310934357

Comment (ruudk) [reply tests/bootstrap.php:9]: It fails on SF 2.8 with PHP 7.2

Comment (garak) [reply tests/bootstrap.php:9]: We could require symony/phpunit-bridge then, just to not reinvent the wheel

**Pull request merged**